### DISCUSSION TITLE
可否添加个隐藏/禁用pcl2更新模组功能的选项

### DISCUSSION THREAD

**Discussion by E0x72-21** [Meloong-Git/PCL#5032] [提问] [answered]
也就是禁用/隐藏这个功能，因为有些整合包会因为用户乱更新模组炸掉
![image](https://github.com/user-attachments/assets/8032e1e4-705d-47b3-a7a0-c5b495c5cfba)

开关可以放在这里（
![image](https://github.com/user-attachments/assets/db125d5f-982b-4191-a3e9-eab80b04db84)

**Comment by tangge233** [comment]
PCL 会在用户首次使用更新 Mod 功能时提示相关信息

~~当然，用户很不听警告~~

**Reply by E0x72-21** [reply]
~~整合包作者表示很淦~~

**Comment by LTCatt** [comment, marked as answer]
可以，交个 issue 吧
